apps/omg_child_chain_rpc/priv/swagger/operator_api_specs/block/request_bodies.yaml

```diff
@@ -4,6 +4,7 @@ GetBlockBodySchema:
   content:
     application/json:
       schema:
+        title: 'block_parameter'
```

Comment:
It's not `parameter` but full body (here contains just one field).
Also: why not make it more predictable e.g. `get_block_body_schema` to follow the element key (`GetBlockBodySchema`) or even better keep it the same `title: 'GetBlockBodySchema'`